### ISSUE TITLE
Kibana page shows error

### ISSUE THREAD

**Issue by DigiAngel** [dunbarcyber/cyphon#183]
So going to:
~~~
http://ip:5601/app/kibana
~~~

gives this:
~~~
{"statusCode":500,"error":"Internal Server Error","message":"An internal server error occurred"}
~~~

Haven't made any changes yet....what's my first step to troubleshoot?  Thank you.

**Comment by darvid** [comment]
I've gotten this issue sporadically when starting the docker containers, I think it's related to startup priority. Could you try running the following commands in the VM, assuming you're logged in and in the ```/opt/cyphondock``` directory?

```
$ docker-compose restart elasticsearch
$ docker-compose restart kibana
$ docker-compose restart nginx
```

This usually fixes it for me, but if it's still not working, maybe try grabbing the logs off the Kibana container, with ```docker-compose logs -f kibana``` (in the cyphondock directory).

**Comment by DigiAngel** [comment]
Tried the first restart with elasticsearch but here's what I got:
~~~
ERROR: for cyphondock_elasticsearch_1  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)
ERROR: An HTTP request took too long to complete. Retry with --verbose to obtain debug information.
If you encounter this issue regularly because of slow network conditions, consider setting COMPOSE_HTTP_TIMEOUT to a higher value (current value: 60).
~~~

Additionally, this image ran at 100% all weekend with no data going to it at all:
![2017-08-14 11_44_22-cyphon](https://user-images.githubusercontent.com/947191/29283930-010df0e2-80e6-11e7-9465-7d2277c21b29.png)

I will close this out and wait for an image/ova that isn't using docker.  Thank you.
